doc/classes/PackedByteArray.xml

```diff
@@ -479,6 +487,33 @@
 				If the original data can't be converted to signed 64-bit integers, the resulting data is undefined.
 			</description>
 		</method>
+		<method name="to_vector2_array" qualifiers="const">
+			<return type="PackedVector2Array" />
+			<description>
+				Returns a copy of the data converted to a [PackedVector2Array], where each block of 8 bytes has been converted to a [Vector2] variant.
+				The size of the input array must be a multiple of 8 (size of two 32-bit float variables). The size of the new array will be [code]byte_array.size() / 8[/code].
```

Comment:
The note about double precision should also be made here IMO, the 8 bytes doesn't apply for double builds and there's a risk of missing the note below

Reply:
Sure, I rephrased it a bit to include that.